tests/endtoend/opart/comparison/forward_time_opart.py

```diff
@@ -0,0 +1,86 @@
+from exojax.spec.opacalc import OpaPremodit
+from exojax.spec.opart import OpartEmisPure
+from exojax.spec.layeropacity import single_layer_optical_depth
+from exojax.utils.grids import wavenumber_grid
+from exojax.spec.api import MdbExomol
+from exojax.utils.astrofunc import gravity_jupiter
+import time
+import tqdm
+import jax.profiler
+import jax.numpy as jnp
+from jax import config
+config.update("jax_enable_x64", True)
+from jax_smi import initialise_tracking
+initialise_tracking()
+
+class OpaLayer:
+    # user defined class, needs to define self.nugrid
+    def __init__(self, Nnus=100000):
+        self.nu_grid, self.wav, self.resolution = wavenumber_grid(
+            1900.0, 2300.0, Nnus, unit="cm-1", xsmode="premodit"
+            #2050.0, 2150.0, Nnus, unit="cm-1", xsmode="premodit"
+
+        )
+        self.mdb_co = MdbExomol(".database/CO/12C-16O/Li2015", nurange=self.nu_grid)
+        self.opa_co = OpaPremodit(
+            self.mdb_co,
+            self.nu_grid,
+            auto_trange=[500.0, 1500.0],
+            dit_grid_resolution=1.0,
+            allow_32bit=True
```

Comment:
```suggestion
```
Would it be better to use the 64-bit so that the opart calculation is consistent with forward_time_opa_and_art.py?